# DataScience_DeepLearning_MachineLearning
The issue
Getting the message "Sorry, something went wrong. Reload?" when viewing an *.ipynb on a GitHub blob page.

Probably a problem with the GitHub notebook viewing tool - sometimes github fails to render the ipynb notebooks, I believe that is some temporary (and recurring) issue with their backend
A workaround
Try to open that notebook that you want using nbviewer online, you don't need to install it.

Open "https://nbviewer.jupyter.org/"
Paste the link to your notebook, (e.g. "https://github.com/KumarVishwesh1992/DataScience_DeepLearning_MachineLearning/blob/master/Classification/DecisionTree.ipynb") there and then you get 
Ref: https://github.com/iurisegtovich/PyTherm-applied-thermodynamics/issues/11
